src/System.Runtime.Intrinsics/src/ApiCompatBaseline.netcoreappaot.txt

```diff
@@ -1,89 +1,9 @@
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector128(System.Runtime.Intrinsics.Vector128<System.UInt16>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector128(System.Runtime.Intrinsics.Vector128<System.UInt32>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector128(System.Runtime.Intrinsics.Vector128<System.UInt64>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Byte>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Double>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Int16>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Int32>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Int64>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.SByte>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.Single>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.UInt16>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.UInt32>)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Runtime.Intrinsics.X86.Avx2.BroadcastScalarToVector256(System.Runtime.Intrinsics.Vector128<System.UInt64>)' does not exist in the implementation but it does exist in the contract.
+warning : Did not find type 'T:System.Runtime.Intrinsics.X86.Bmi1.X64' in any of the seed assemblies.
```

Comment:
I'm not sure we need this. I had seen it in my PR, initially as well. But it appears to have gone away after rebasing onto the current HEAD.

Reply:
(https://github.com/dotnet/corefx/pull/33302)

Reply:
According to my experience, we need this for "Packaging All Configurations x64 Debug Build"

Reply:
Looking at the binlog, the seed assemblies being passed in are:
```
C:\Users\tagoo\.nuget\packages\microsoft.targetingpack.private.netnative\1.1.0-beta-27106-00\lib\uap10.1\System.Private.CoreLib.dll
E:\Users\tagoo\Repos\corefx\artifacts\bin\ref/netcoreapp/netstandard.dll
```

Reply:
Ok, this looks like just a case of the relevant dependencies haven't updated yet.

Reply:
Most of the time for net native we just baseline it and clean up later.

Reply:
@fiigii, the .NET Native dependencies have been updated. This should pass without ApiCompatBaseline changes now (although we can still probably remove the ones that are no longer applicable).

Reply:
Hmmm, actually; even the updated package still doesn't have the new types ☹️, so we might still need the changes after all.
* You can validate that the changes pass locally with `.\build.cmd -allConfigurations`

@ericstj, do you know what is required to get it to ignore the `Did not find type 'T:System.Runtime.Intrinsics...' in any of the seed assemblies.` messages? Just adding the exact text doesn't appear to work.

Reply:
Is this a scenario that is only handled by `ignoreMissingTypes`?
* It would also be good to find out why the uap corelib isn't updated here.... These are part of the shared types and have been checked in on the CoreCLR side for a few days now